packages/keystone/fetch.js

```diff
@@ -1,14 +1,41 @@
+const os = require('os')
+
+const { pickBy } = require('lodash')
 const nodeFetch = require('node-fetch')
 
+const conf = require('@open-condo/config')
+
 const { getExecutionContext } = require('./executionContext')
 const { getLogger } = require('./logging')
 const Mertrics = require('./metrics')
 
+
 const logger = getLogger('fetch')
 
 const FETCH_COUNT_METRIC_NAME = 'fetch.count'
 const FETCH_TIME_METRIC_NAME = 'fetch.time'
 
+const HOSTNAME = os.hostname() || ''
+const NAMESPACE = conf.NAMESPACE || ''
+const VERSION = conf.WERF_COMMIT_HASH || ''
+
+const getDeploymentName = () => {
```

Comment:
it's better to use this function for `getAppName`
and take it from `APP_NAME` if it's exists if no use your current logic